DigraphyApi/Utils/OrderFilters.cs

```diff
@@ -0,0 +1,7 @@
+namespace DigraphyApi.Utils;
+
+public class OrderFilters(int? collectionId, bool? verified)
+{
+    public int? CollectionId = collectionId;
+    public bool? Verified = verified;
```

Comment:
These aren't currently properties, they're fields. What you need to do is:
`
```suggestion
    public int? CollectionId { get; } = collectionId;
    public bool? Verified { get; } = verified;
```